lib/matplotlib/__init__.py

```diff
@@ -99,6 +99,8 @@
 to MATLAB&reg;, a registered trademark of The MathWorks, Inc.
 
 """
+# NOTE: This file must remain Python 2 compatible for the forseeable future,
+# to ensure that we error out properly for existing editable installs.
```

Comment:
In that case it may be worth having a test entry that just tries to `python2 -mcompileall matplotlib/__init__.py` (or similar) to check that indeed the file is Py2-compatible.